projects/exui-common-lib/src/lib/components/pagination/pagination.component.scss

```diff
@@ -1,3 +1,3 @@
-span.moj-pagination__link:hover {
+span.hmcts-pagination__link:hover {
   color: revert;
```

Comment:
Are we still developing for Internet Explorer? If so, this is not compatible.

Reply:
Hi @paulhowes-HMCTS - we've been told we no longer need to support IE.